python/datasources/acs_population.py

```diff
@@ -574,3 +577,44 @@ def _create_ingesters(self):
             ACSPopulationIngester(False, BASE_ACS_URL),
             ACSPopulationIngester(True, BASE_ACS_URL)
         ]
+
+
+def generate_national_dataset_with_all_states(state_df, demographic_breakdown_category):
+    all_state_fips = set(state_df[std_col.STATE_FIPS_COL].drop_duplicates().to_list())
```

Comment:
do you need to both cast it to a set and use drop_duplicates() ? isn't that doing the same thing?

Reply:
Good catch, I don't need to drop_duplicates() because turning the list into a set will do that. 

But it *is* important to use a set because the lookup time is O(1), which is what we are using it for.